### ISSUE TITLE
Comms issue #34

### ISSUE THREAD

**Issue by nfee006** [asc-csa/Scisat-App#132]
The update button is a clickable span and it is impossible to put it in focus using the keyboard. It should be a button.

**Comment by emilinecsa** [comment]
This issue is fixed. It works now using the keyboard.
